dashboard/app/models/unit_group.rb

```diff
@@ -33,6 +33,7 @@ class UnitGroup < ApplicationRecord
   has_one :plc_course, class_name: 'Plc::Course', foreign_key: 'course_id'
   has_many :default_unit_group_units, -> {order(:position)}, class_name: 'UnitGroupUnit', dependent: :destroy, foreign_key: 'course_id'
   has_many :default_units, through: :default_unit_group_units, source: :script
+  has_many :original_units, class_name: 'Unit', foreign_key: 'original_unit_group_id'
```

Comment:
I think this would be a good place to document the difference between default_units and original_units. otherwise it is a bit intimidating to see these two lists of similar looking (and sometimes identical) units